cmd/exportResults.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/integr8ly/delorean/pkg/utils"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+type MetaData struct {
+	Name    string	`json:"name"`
+	Version	string	`json:"rhmi-version"`
+	Job		string	`json:"job-link"`
+}
+
+type testResultsCmdFlags struct {
+	testResultsDir 	string
+	pipelineName	string
+	rhmiVersion 	string
+	jobLink			string
+	accessKeyId		string
+	secretAccessKey string
+}
+
+func init() {
+	f := &testResultsCmdFlags{}
+	cmd := &cobra.Command{
+		Use:   "export-results",
+		Short: "Export RHMI product test results to s3 bucket",
```

Comment:
```suggestion
		Short: "Export RHMI and product test results to s3 bucket",
```
we should try to explain that it should export also the RHMI tests and not only the product tests